packages/coin-support-evm/src/operations/broadcastTransaction/index.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  getAccountAndCoin,
+  insertOrUpdateTransactions,
+} from '@cypherock/coin-support-utils';
+import { evmCoinList } from '@cypherock/coins';
+import { BigNumber } from '@cypherock/cysync-utils';
+import {
+  ITransaction,
+  TransactionStatusMap,
+  TransactionTypeMap,
+} from '@cypherock/db-interfaces';
+
+import { IBroadcastEvmTransactionParams } from './types';
+
+import { broadcastTransactionToBlockchain } from '../../services';
+
+export const broadcastTransaction = async (
+  params: IBroadcastEvmTransactionParams,
+): Promise<ITransaction> => {
+  const { db, signedTransaction, transaction } = params;
+  const { account, coin } = await getAccountAndCoin(
+    db,
+    evmCoinList,
+    transaction.accountId,
+  );
+  const isMine =
+    params.transaction.computedData.outputs[0].address ===
+    account.xpubOrAddress;
```

Comment:
Convert to same case when comparing address strings

Reply:
changed